testing/regress/ecl/cppbody11.ecl

```diff
@@ -24,7 +24,7 @@ real scoreIt(scoreFunc func, real a, real b) := DEFINE
     
 real scoreIt2(scoreFunc func, real a, real b) := BEGINC++
     
-    return func(a-1.0,b-1.0) * func(a+1.0,b+1.0);
```

Comment:
Yes you are right.  I started off adding a #option context, but then there were problems with prototypes of user defined functions.  In the end I went for the option where all functions are assumed to have a context - unless they are external c++ functions without the context attribute.